### PR TITLE
Relationships

### PR DESCRIPTION
**Pull request by kemigabocatherine**
### Changes made

- Added new files;
     1. book.rb
     2. rental.rb
     3. classroom.rb

- Added has many relationships between book and rental, and person and rental
- Added belongs-to relationships between rental and book, and rental and person
- Added a many-to-many relationship between rental, book, and person

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```
Comment (dasileker) [inline book.rb:5]: - [ ] optional the `rentals` is also should access the book

File: person.rb [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
+    @rentals = []
   end
 
   def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```
Comment (dasileker) [inline person.rb]: - [ ] the` add-rental` should take **date** and **book** as parameters and also should be passed to the new rental every time you add a new book.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @kemigabocatherine,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**

- Linters are good ✅.
- The commit is nice ✅.
- The readme file is good ✅.
- The PR is good ✅.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @kemigabocatherine  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**